server/server_test.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	testTimeout            = 10 * time.Second
+	testTimeout            = 5 * time.Second
```

Comment:
it won't work in all times on CI, really ;)

Reply:
Ouch, I forgot to increase it again. I'll remove this change